app/services/notify_service.rb

```diff
@@ -65,7 +65,12 @@ def create_notification
   end
 
   def send_push_notifications
-    WebPushNotificationWorker.perform_async(@recipient.id, @notification.id)
+    sessions_with_subscriptions = @recipient.user.session_activations.where.not(web_push_subscription: nil)
+
+    Sidekiq::Client.push_bulk(
+      'class' => WebPushNotificationWorker,
+      'args' => sessions_with_subscriptions.map { |session_activation| [session_activation.id, @notification.id] }
```

Comment:
Oh, that implementation is actually not really good. You can do something like this:
```Ruby
sessions_with_subscriptions.pluck(:id).map { |session_activation_id| [session_activation_id, @notification.id] }
```